src/realm-settings/keys/KeysListTab.tsx

```diff
@@ -55,81 +57,35 @@ export const KeysListTab = ({ realmComponents }: KeysListTabProps) => {
     });
 
     const keys = keysMetaData.keys;
+    const filtered =
+      filterType !== FILTER_OPTIONS[0]
+        ? keys?.filter((i) => i.status === filterType)
```

Comment:
You can get rid of the single letter variable if you use destructuring here.
```suggestion
        ? keys?.filter(({ status }) => status === filterType)
```